docs/setup/postinstall.rst

```diff
@@ -11,7 +11,7 @@ After you connect to your Vagrant box, make sure you are in the ``intranet`` dir
 
 You will then need to run ``python manage.py migrate`` to set up the Postgres database.
 
-You can then start the built-in Django web server with ``fab runserver``. Now that you are running the development server, open a browser to http://127.0.0.1:8080 and log in (the default master password is ``swordfish``). If it fails, check the output of ``python manage.py runserver``.
+You can then start the built-in Django web server with ``fab runserver``. Now that you are running the development server, open a browser to http://127.0.0.1:8080 and log in (the default master password is ``swordfish``). Make sure to create an user by doing the steps in the following section in order to login. If it still fails, check the output of ``python manage.py runserver``.
```

Comment:
This was covered in #976. Please remove this set of changes; they will only cause conflicts.